packages/components/src/components/Icon/Icon.tsx

```diff
@@ -205,3 +205,5 @@ export const Icon = forwardRef(
         );
     },
 );
+
+export const Icon = memo(IconComponent);
```

Comment:
Is this a good solution? It seems little bit like treating the symptoms and not the underlying issue:)) Why not wrap every component in memo?